spec/requests/spree/api/v2/storefront/account_confirmation_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'spec_helper'
+
+describe 'Storefront API v2 Account Confirmation spec', type: :request do
+  describe 'account_confirmations#show' do
+
+    before do
+      set_confirmable_option(true)
+      get "/api/v2/storefront/account_confirmations/#{confirmation_token}"
+    end
+
+    after(:each) { set_confirmable_option(false) }
+
```

Comment:
I am still trying to fix failing tests.
It seems like the above blocks have an impact on other spec files.